src/Workspaces/Core/Portable/Workspace/Solution/TextDocumentState.cs

```diff
@@ -237,12 +287,18 @@ public bool TryGetTextVersion(out VersionStamp version)
             return textAndVersion.Text;
         }
 
-        public SourceText GetText(CancellationToken cancellationToken)
+        public SourceText GetTextSync(CancellationToken cancellationToken)
```

Comment:
We've been using the naming pattern of "GetBlahSynchronously".  Can we keep that pattern?